rust/pact_verifier/src/messages.rs

```diff
@@ -119,3 +121,29 @@ fn display_result(body_result: ANSIGenericString<str>, metadata_result: Vec<(Str
   }
   println!("      has a matching body ({})", body_result);
 }
+
+fn extract_metadata(actual_response: &Response) -> HashMap<String, String>{
+  let mut metadata = hashmap!{
+    "contentType".into() => actual_response.lookup_content_type().unwrap_or_default(),
+    "Content-Type".into() => actual_response.lookup_content_type().unwrap_or_default()
```

Comment:
Just use `contentType` here

Reply:
OK. I meant to raise that in the request as well. Can we assume that any consumer adding `Content-Type` as an expectation is satisfied by this?